packages/core-utils/src/route.ts

```diff
@@ -390,7 +412,9 @@ function makeMultiCriteriaSort(...criteria) {
  *    those with shortNames.
  *  7. longName as string.
  */
-export function makeRouteComparator(transitOperators) {
+export function makeRouteComparator(
+  transitOperators: TransitOperator[]
+): (a: unknown, b: unknown) => number {
```

Comment:
Aren't `a` and `b` of type `Route`?

Reply:
They are `number`s. Good catch thank you!